### PR TITLE
Grass Cure % Position Fix

### PR DESCRIPTION
**Pull request by JulianForeman**
* Julian moved position of grass cure col over so it's next to fuel type for both daily & week table views
* I snuck in a bunch of other little UI changes that have been bugging me for months. Broke the rules of Agile but #YOLO. Specifically:
    * no **more random** bolding of **some column headers** and n**ot oth**ers. Every col header is bold now
    * Fixed the stickyCell/cell border interaction jank - everything should be nicely vertically aligned now, and no more double border for the first day in weekly view
    * Made one component for common headers, shared between weekly & daily tables
---------

# Test Links:
[Percentile Calculator](https://wps-pr-1852.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-1852.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-1852.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-1852.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-1852.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=1108&f=c5&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN#state=2ec784ca-c46a-49d0-b2b3-1cf32a9015a2&session_state=7d9447c8-db66-4661-b4cb-03d2ac0d1d8f&code=32292df4-2bdf-4f90-a4a8-c8dbcda682a9.7d9447c8-db66-4661-b4cb-03d2ac0d1d8f.2b63f390-f3dc-43ae-89f2-016453863476)
[Fire Behaviour Advisory](https://wps-pr-1852.apps.silver.devops.gov.bc.ca/fire-behaviour-advisory)
[HFI Calculator](https://wps-pr-1852.apps.silver.devops.gov.bc.ca/hfi-calculator)
[FWI Calculator](https://wps-pr-1852.apps.silver.devops.gov.bc.ca/fwi-calculator)

### CODE REVIEW

**Review by conbrad — CHANGES_REQUESTED** [review]
Nice, just need to also add the change to the PDF generation

**Comment by JulianForeman** [comment]
> Member

It looks like you already put grass cure next to fuel type in your template

**Comment by JulianForeman** [comment]
> Nice, just need to also add the change to the PDF generation

I think you handled this in the pdf generation because we worked from an accurate mockup, not the app.

**Comment by conbrad** [comment]
> > Nice, just need to also add the change to the PDF generation
> 
> I think you handled this in the pdf generation because we worked from an accurate mockup, not the app.

I think you're right, in that case LGTM

File: web/src/features/hfiCalculator/components/DailyViewTable.tsx [hunk 1]
```diff
@@ -175,6 +175,13 @@ export const DailyViewTable = (props: Props): JSX.Element => {
                     <br />
                     Type
                   </TableCell>
+                  <TableCell>
```
Comment (conbrad) [inline web/src/features/hfiCalculator/components/DailyViewTable.tsx:178]: This will need to be in it's own `StickCell` > `Table` tree

**Review by conbrad — CHANGES_REQUESTED** [review]

**Review by tessmgood — COMMENTED** [review]
Placement looking good - there is a weird styling gremlin now, lmk if it's quick to tidy:

There is a grey line under grass cure in daily tabs - but not in prep view. Seems like a cell outline or underline? it's visual noise I'd love to remove if possiblee
Screen Shot 2022-04-05 at 3 31 42 PM

**Comment by tessmgood** [comment]
Something has happened to our columns in daily tabs - is danger class label missing? Fire type onwards is offset 
Screen Shot 2022-04-05 at 3 35 28 PM

**Comment by andrea-williams** [comment]
> Something has happened to our columns in daily tabs - is a label missing? HFI onwards is offset Screen Shot 2022-04-05 at 3 35 28 PM

@tessmgood  you peeked too early! Still working on it, although I should have changes ready for you to review soon

**Comment by tessmgood** [comment]
> > Something has happened to our columns in daily tabs - is a label missing? HFI onwards is offset Screen Shot 2022-04-05 at 3 35 28 PM
> 
> @tessmgood you peeked too early! Still working on it, although I should have changes ready for you to review soon

oop - I think Julian tagged me for review yesterday, my b!

**Review by conbrad — APPROVED** [review]
Nice! Thanks for fixing that issue while you were working on the code!

File: web/src/features/hfiCalculator/components/StaticCells.tsx [hunk 2]
```diff
@@ -28,6 +28,10 @@ export const StaticCells = ({
   const staticCells = range(numPrepDays).map(dailyIndex => {
     const daily = dailies?.at(dailyIndex)
     const error = !isValidGrassCure(daily, station.station_props)
+    let isFirstDayOfPrep = false
```
[the remark this answers is not in the document]
Comment (andrea-williams) [reply web/src/features/hfiCalculator/components/StaticCells.tsx]: Thanks :) Made it inline in prop param

**Review by tessmgood — APPROVED** [review]
Looks great and rule-breaking is 🦄 🌈

**Review by Sybrand — APPROVED** [review]

**Pull request merged**